picard/ui/options/interface.py

```diff
@@ -199,8 +207,10 @@ def fcmp(x):
         self.move_view = MoveableListView(self.ui.toolbar_layout_list, self.ui.up_button,
                                           self.ui.down_button, self.update_action_buttons)
         self.update_buttons = self.move_view.update_buttons
-        if IS_MACOS or IS_WIN or IS_HAIKU:
-            self.ui.use_system_theme.hide()
+
+        if not OS_SUPPORTS_THEMES:
+            self.ui.ui_theme.hide()
+            self.ui.label_theme.hide()
```

Comment:
Minor UI issue: The ui_theme combo box is inside a layout container (currently called `horizontalLayout_2`) together with an additional spacer. Hiding just the combobox still leaves this to use space because of this spacer.

I would suggest changing this by adding a plain empty Widget, name it something like `ui_theme_container` and make the current label and combobox + spacer childs of this container widget. The container should have layout margins all set to 0 to not introduce additional spacing around it. Then you can hide just this container here:

    self.ui.ui_theme_container.hide()

Reply:
No idea how to do that with no margins and still look right. It refused to align with the other stuff in the same layout.

Reply:
Yeah, Qt can be tricky here ;) I pushed https://github.com/metabrainz/picard/pull/1761/commits/bc908c41ec1310529dfda90e179bc0ac2aaed519 to your branch.  You can just squash my commit into the previous one.

The trick is to not add a layout as a child to the container widget, but have the widget itself have a vertical layout. In Qt designer if you right click on a widget that has childs you can select an explicit layout for this container. This only works if there is at least one child already, that can sometimes be annoying.